### ISSUE TITLE
session: Reimplement `mc session` sub-commands to use cli sub-command style

### ISSUE THREAD

**Issue by harshavardhana** [minio/mc#2211]
## Expected behaviour
mc session when typed should print in the same manner like `mc admin` or `mc config host` prints their sub-command and each sub-command should document itself. 

## Actual behaviour

Currently it can seen that it is harder to document our help text below which provides confusing documentation.

```
$ mc session 
NAME:
  mc session - Manage saved sessions for cp and mirror commands.

USAGE:
  mc session [FLAGS] OPERATION [SESSION-ID]

OPERATION:
  resume   Resume a previously saved session.
  clear    Clear a previously saved session.
  list     List all previously saved sessions.

SESSION-ID:
  SESSION - Session can either be $SESSION-ID or "all".

FLAGS:
  --config-folder value, -C value  Path to configuration folder. (default: "/home/harsha/.mc")
  --quiet, -q                      Suppress chatty console output.
  --no-color                       Disable color theme.
  --json                           Enable JSON formatted output.
  --debug                          Enable debug output.
  --insecure                       Disable SSL certificate verification.
  --help, -h                       Show help.
  
EXAMPLES:
   1. List sessions.
      $ mc session list

   2. Resume session.
      $ mc session resume ygVIpSJs

   3. Clear session.
      $ mc session clear ygVIpSJs

   4. Clear all sessions.
      $ mc session clear all
```

## Steps to reproduce the behaviour
Just type `mc session`

## mc version
master

## System information
All operating systems

**Comment by awwalker** [comment]
@deekoder @harshavardhana if no one is actively working on this I can give it a go...

**Comment by harshavardhana** [comment]
Fixed by #2225
